### PR TITLE
Update snyk check

### PR DESCRIPTION
**Pull request by MartinBasti**

### CODE REVIEW

File: task/sast-snyk-check/0.1/sast-snyk-check.yaml [hunk 1]
```diff
@@ -48,8 +49,7 @@ spec:
         SKIP_MSG="We found 0 supported files"
         grep -q "$SKIP_MSG" stdout.txt || test_not_skipped=$?
 
-        HACBS_ERROR_OUTPUT=$(jq -rc --arg date $(date +%s) --null-input \
-          '{result: "ERROR", timestamp: $date}')
+        HACBS_ERROR_OUTPUT=$(make_result_json -r ERROR)
```
Comment (hongweiliu17) [inline task/sast-snyk-check/0.1/sast-snyk-check.yaml:52]:
```suggestion
        HACBS_ERROR_OUTPUT=$(make_result_json -r ERROR -f 1)
```

Comment (hongweiliu17) [reply task/sast-snyk-check/0.1/sast-snyk-check.yaml:52]: Or, sorry, we don't need a number for `ERROR` result, please ignore this comment.

**Review by hongweiliu17 — APPROVED** [review]
LGTM

**Review by srivickynesh — APPROVED** [review]
LGTM

**Pull request merged**